ksqldb-functional-tests/src/test/resources/query-validation-tests/fk-join.json

```diff
@@ -1,5 +1,23 @@
 {
   "tests": [
+    {
+      "name": "Inner join",
```

Comment:
This one passed now 🎉

Reply:
Hooray!! Great stuff. What do you think about getting https://github.com/confluentinc/ksql/pull/7537 merged first and simply enabling select tests from that PR, rather than introducing new tests in this PR? That way we can avoid merge conflicts / duplicate tests.

The PR build passed on https://github.com/confluentinc/ksql/pull/7537 already. I still have more cleanup to do on the PR but we can merge first to unblock you and I can open a follow-up for the cleanup.